client/src/App.jsx

```diff
@@ -0,0 +1,25 @@
+import Navbar from './Navbar.jsx';
+import EntryList from './EntryList.jsx';
+import EditEntry from './EditEntry.jsx';
+import { useState } from 'react';
+import { data } from './data.js';
+
+function App() {
+  const [view, setView] = useState('edit-entry');
+
+  function handleEdit() {
+   setView('edit-entry')
+  }
+
+  return (
+    <div>
+      <Navbar onClick={() => setView('entries')} />
+      {view === 'entries' && (
+        <EntryList onClick={() => setView('edit-entry')} data={data} onEdit={handleEdit}/>
+      )}
+      {view === 'edit-entry' && <EditEntry />}
```

Comment:
I see you didn't get to Edit, but when you do, you'll need to pass the entry being edited to the EditEntry component.